charts/airflow/values.yaml

```diff
@@ -1080,6 +1080,13 @@ dags:
     ##
     sshKnownHosts: ""
 
+    ## the number of consecutive failures allowed before aborting
+    ##
+    ## NOTE:
+    ##  - the first sync must succeed
+    ##  - a value of -1 will retry forever after the initial sync
```

Comment:
@n-oden I am super sorry to be tedious, but can you add the `##` at the bottom of the docstring.

For example,
```
    ## the number of consecutive failures allowed before aborting
    ##
    ## NOTE:
    ##  - the first sync must succeed
    ##  - a value of -1 will retry forever after the initial sync
    ##
    maxFailures: 0
```

Reply:
trust me I am the last person in the world to object to fussy formatting feedback 😂 

Updated!